.irbrc

```diff
@@ -1,6 +1,5 @@
-$:.unshift File.expand_path('lib')
+$LOAD_PATH.unshift File.expand_path('lib')
 
 require 'webflow'
 
-TEST_API_TOKEN = '1f0da5c9368af9cb2dcd65d22a6600a8ffa069f70729e129a09787203bc2c2be'
```

Comment:
This is intentional, dont remove the test token. Use it as a fallback throughout.

Reply:
I couldn't get a single API query to return anything other than:
> /Users/viktor/Developer/gems/webflow-ruby/lib/webflow/client.rb:152:in `request': Request not authorized (Webflow::Error)

That's why I left it to the user (developer) to pass in their own working API token.

Reply:
hm, looks like it has vanished... 🤷  at least i fixed my blog while looking for it 😆